### ISSUE TITLE
Payment Details page - force focus cancel button

### ISSUE THREAD

**Issue by KateChuen** [SAP/cloud-commerce-spartacus-storefront#6511]
**Environment Details**
Spartacus: epic/accessibility branch
Browser: Chrome & Safari
Device: mac
Happens on: http://dev-com-3.accdemo.b2c.ydev.hybris.com:4200/electronics-spa/en/USD

**Steps to Reproduce**
- Go to MyAccount -> Payment Details Page
- Have at least 2 payments methods
- Tab to the default payment method, and press Enter on "Delete" button.
- Tab -> You'll see that the next tab goes to the second payment card -> Not a big deal, since we can tab back to the first card.

<img width="1187" alt="Screen Shot 2020-02-12 at 10 53 53 AM" src="https://user-images.githubusercontent.com/45409515/74352071-fd119b00-4d85-11ea-956b-88903695c065.png">

But more of a consistent issue. Because on firefox, it works more intuitively. The next tab puts the focus on the "Cancel" key on the first payment instead. 

<img width="1195" alt="Screen Shot 2020-02-12 at 10 53 45 AM" src="https://user-images.githubusercontent.com/45409515/74352085-03077c00-4d86-11ea-8feb-9c9b2750a158.png">

**Comment by kacperknapik** [comment]
### Proposed solution
After pressing enter on `delete` button, force focus the `cancel` button

**Comment by KateChuen** [comment]
Please also check on Address Book Page as well. After finishing editing an address, as well as deleting an address.

**Comment by mszczepaniak** [comment]
**QA Steps:**

1. Go to MyAccount -> Payment Details Page
2. You should have at least one payment data saved.
3. Press Tab to the default payment method, and press Enter on `Delete` button.
4. You should have focus on the `Cancel` button.

**Comment by Xymmer** [comment]
Looks like this ticket was overlooked
There's a PR from Lohit: #7355
Moving to code review

**Comment by Xymmer** [comment]
@kacperknapik we discovered this issue without milestone, there seems to be a pr, can you validate?
i set to must but if not must, set to 2.x or 2.1

**Comment by kacperknapik** [comment]
This issue is still open, because the PR didn't fix the problem.

**Comment by psmul** [comment]
After talking with @tobi-or-not-tobi, there's a decision that this issue should be postponed in time due to possible card component refactor.

**Comment by Xymmer** [comment]
setting to onhold until refactoring done

**Comment by giancorderoortiz** [comment]
Recheck if still an issue

**Comment by giancorderoortiz** [comment]
Still an issue. Rechecked with 4.3.0. 
See http://spartacus-devci7672.eastus.cloudapp.azure.com:4200/electronics-spa/en/USD/my-account/payment-details
